cypress/support/commands/analytics.js

```diff
@@ -29,8 +29,8 @@ Cypress.Commands.add('hasGlobalChartbeatConfig', () => {
 });
 
 // Should be moved into integration/pages/index.js once all pages have Chartbeat
-Cypress.Commands.add('hasAmpChartbeatConfigUid', () => {
+Cypress.Commands.add('hasAmpChartbeatConfigUid', (position = 1) => {
   cy.get('amp-analytics script[type="application/json"]')
-    .eq(1)
+    .eq(position)
```

Comment:
Do we need this`eq` at all?  @karinathomasbbc had a similar issue with script tags being in different positions - it seemed to work fine with the `.eq` removed:

https://github.com/bbc/simorgh/commit/f399feb3cb1d05ed545b4fe110763202e990e770

Reply:
Removed :)

Reply:
👍